src/main/java/com/openshift/restclient/model/volume/property/IPersistentVolumeProperties.java

```diff
@@ -0,0 +1,10 @@
+package com.openshift.restclient.model.volume.property;
+
+
+import org.jboss.dmr.ModelNode;
+
+public interface IPersistentVolumeProperties {
```

Comment:
I decided to go with entirely new interface instead of using IVolume, because PersistentVolume is type of Volume as a whole. But persistent volume Type are just property values for PersistentVolume.